Rebus.AmazonSQS/AmazonSQSTransport.cs

```diff
@@ -383,17 +383,32 @@ private TransportMessage GetTransportMessage(Message message)
         {
             var headers = message.MessageAttributes.ToDictionary(kv => kv.Key, kv => kv.Value.StringValue);
 
+            if (!headers.ContainsKey(Headers.ContentType))
+                headers[Headers.ContentType] = "application/json;charset=utf-8";
+
+            if (!headers.ContainsKey(Headers.ReturnAddress))
+                headers[Headers.ReturnAddress] = Address;
+
+            if (!headers.ContainsKey(Headers.MessageId))
+                headers[Headers.MessageId] = message.MessageId;
+
+            if (!headers.ContainsKey(Headers.CorrelationId))
+                headers[Headers.CorrelationId] = message.MessageId;
+
+            if (!headers.ContainsKey(Headers.Intent))
+                headers[Headers.Intent] = Headers.IntentOptions.PublishSubscribe;
+
             return new TransportMessage(headers, GetBodyBytes(message.Body));
 
         }
         private string GetBody(byte[] bodyBytes)
         {
-            return Convert.ToBase64String(bodyBytes);
+            return System.Text.Encoding.UTF8.GetString(bodyBytes);
```

Comment:
Sorry, but this will not work in the general case.... it is not guaranteed that the configured serializer is a JSON serializer, and even if it was a JSON serializer the transport should not assume UTF8 encoding

Is there no way for Amazon SQS to send raw bytes?